task/buildah/0.1/buildah.yaml

```diff
@@ -92,7 +92,7 @@ spec:
         cpu: 2
       requests:
         memory: 512Mi
-        cpu: 10m
+        cpu: 250m
```

Comment:
In a bit of cross referencing with other tekton based projects that use builds, like the shipwright project that @sbose78 @adambkaplan and I used to be more involved with, they are also setting memory reqeusts to 250m

See https://github.com/shipwright-io/build/blob/f76244975148458a4afb4b3bd2d855710c29281a/samples/buildstrategy/buildah/buildstrategy_buildah_shipwright_managed_push_cr.yaml#L175

IBM's used of shipwright, with tekton under the covers, is also a similar service based offering which they have been running at scale.

Hence I would consider this value vetted even beyond what @amfred and the performance folks saw today.

And when comparing 250m vs. the 2 full core limit, that is still  a conservative start IMO.

Let's go with this.